### DISCUSSION TITLE
The WeasyPrint Matrix Robots Project is Born

### DISCUSSION THREAD

**Discussion by FrancoisCapon** [FrancoisCapon/TheWeasyPrintMatrixRobotsProject#1] [Announcements]
:gb: Hi,

You can leave your feedback and comments about the project on this discussion.
Sorry for the README.me only in french, I will translate it later (it's a difficult task for me!).

:fr:  Bonjour,

Vous pouvez laisser vos retours et commentaires concernant le projet sur cette discussion.

**Comment by liZe** [comment]
Hi!

That’s an interesting project! It’s always hard to handle inheritance for HTML, and having a kind of matrix handling the modularity is nice to have. A lot of projects exist to manage documents, and some of them are based on HTML, but none is widely used as far as I know and there’s a lot of ideas waiting to be implemented on this topic.

For the HTML generation part, you can also rely on templating libraries (such as Jinja2) that will bring you a lot of useful features for free (like HTML escaping, variables, loops, etc).

Good luck for this project, I hope that you’ll find the time to develop it and get nice users!

**Reply by FrancoisCapon** [reply]
Thanks for your encouragement!
